code/parse/sexp.cpp

```diff
@@ -11981,33 +11982,43 @@ void sexp_cap_waypoint_speed(int n)
 
 /**
  * Causes a ship to jettison its cargo
- * note that the 2nd arg (jettison delay) is not implemented
  */
-void sexp_jettison_cargo(int n)
+void sexp_jettison_cargo(int n, bool jettison_new)
 {
 	char *shipname;
 	int ship_index;
+	int jettison_delay_or_speed;
```

Comment:
If this needs to be passed as a `float`, wouldn't it make more sense to store it as a `float`?